models/cryptopunks/ethereum/cryptopunks_ethereum_punk_transfers.sql

```diff
@@ -0,0 +1,10048 @@
+        , ('0x996737253bc72d0196f5d4110d679b0566dc3e84',4128)
+        , ('0xe45df57df7980d8219a5ea811aff949907d0a355',4129)
+        , ('0xe45df57df7980d8219a5ea811aff949907d0a355',4130)
+        , ('0x00bd9fd57c423a1b1c969823d409156d90974d77',4131)
+        , ('0x00bd9fd57c423a1b1c969823d409156d90974d77',4132)
+        , ('0x00bd9fd57c423a1b1c969823d409156d90974d77',4133)
+        , ('0x1335242f595f7da21ba9dd5cf2bd75d143be6a30',4134)
+        , ('0x00d7c902fbbcd3c9db2da80a439c94486c50eb81',4135)
+        , ('0xe0b432036739a7e98af5af8a2bf307ac02eddd70',4136)
+        , ('0x58164b2d74c45f06cde451fc307092d55ebe5730',4137)
+        , ('0x1335242f595f7da21ba9dd5cf2bd75d143be6a30',4138)
+        , ('0xc24f574d6853f6f6a31c19d468a8c1b3f31c0e54',4139)
+        , ('0x0b30fd3a500608413837bb025c2018933130f9db',4140)
+        , ('0xc24f574d6853f6f6a31c19d468a8c1b3f31c0e54',4141)
+        , ('0x670d7542947f0dd01342d5b3cdbe2b818f8908ea',4142)
+        , ('0xc00c2bcb33f0600ee5466f0f7424ce4a7150e7c8',4143)
+        , ('0xc24f574d6853f6f6a31c19d468a8c1b3f31c0e54',4144)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4145)
+        , ('0x8234be55f64ea91960b78357906cf5bebce81f3d',4146)
+        , ('0x5b098b00621eda6a96b7a476220661ad265f083f',4147)
+        , ('0x5b098b00621eda6a96b7a476220661ad265f083f',4148)
+        , ('0x5b098b00621eda6a96b7a476220661ad265f083f',4149)
+        , ('0xc24f574d6853f6f6a31c19d468a8c1b3f31c0e54',4150)
+        , ('0x23170777466de6046ff42bd5abcf525077e3ce27',4151)
+        , ('0x00d7c902fbbcd3c9db2da80a439c94486c50eb81',4152)
+        , ('0x00dab6b11fa446adaa0cbd4ccdb1626c47e39ab8',4153)
+        , ('0x470e9bd486ebf96ddf6b38de58f21b0872f85776',4154)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4155)
+        , ('0x63e776c2b49b46a2bbc00c31fb5073051b2d6172',4156)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4157)
+        , ('0x02751f403279370dd20a50effb915802b844dd67',4158)
+        , ('0x00bd9fd57c423a1b1c969823d409156d90974d77',4159)
+        , ('0x00000217d2795f1da57e392d2a5bc87125baa38d',4160)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4161)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4162)
+        , ('0x0b30fd3a500608413837bb025c2018933130f9db',4163)
+        , ('0x0b30fd3a500608413837bb025c2018933130f9db',4164)
+        , ('0x809a31ce595288ef3ee7f4ef5a717fc463dcdc0d',4165)
+        , ('0x23170777466de6046ff42bd5abcf525077e3ce27',4166)
+        , ('0xf138eb67f008da11daba9bcffc7a21f31b8b2b14',4167)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4168)
+        , ('0xc272ed09c636edb6e858fa8702c182406185f1af',4169)
+        , ('0xf1ad946ae30a285d96ea396acd3fd22c5aa75ef8',4170)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4171)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4172)
+        , ('0xd4fa6e82c77716fa1ef7f5defc5fd6eeefbd3bff',4173)
+        , ('0x53ede7cae3eb6a7d11429fe589c0278c9acbe21a',4174)
+        , ('0
```

Comment:
consider casting null as datatype, or it'll be a weird 'void' datatype that can cause issues

Reply:
Sorry which null should I cast as datatypes? Just double checked every address and id in this cte is filled in so there won't be any null values

Reply:
sorry, not the cte, but this line: ```select  NULL as from``` -- when you force null into a column, databricks infers an odd datatype. i'd just cast it like: ```cast(NULL as varchar(5))``` or something similar

Reply:
Ah got it - just fixed! Thank you!